src/Escher_Synth.ml

```diff
@@ -154,6 +186,10 @@ let solve_impl ?ast:(ast=false) task consts =
               in ((("0", (fun ars -> vzero)), Const vzero), Array.create ~len:vector_size vzero) in
   let bone = let vone = Th_LIA.vone
              in ((("1", (fun ars -> vone)), Const vone), Array.create ~len:vector_size vone) in
+
+  let nil = let vlist = VList ([],TVar "fresh")
```

Comment:
1. Move this to `Th_List`.
2. Rename to `vempty`.
3. Use `Th_List.vempty` here.